### PR TITLE
Quest Pro controllers for the Interstitial widgets 

### LINKED ISSUE #446: [OpenXR] Add Meta Quest Pro controllers images to the WebXR interstitial widget
We need to generate some screenshots from the 3D models and add them to the repo.

### PR DESCRIPTION
**Pull request by javifernandez**
The WebXRInterstitialWidget creates now controllers for the different Quest devices, including the MetaQuestPro device type. 

The WebXRInterstitailController widget is the one setting OculustQuest for Quest, Quest2 and Pro, since these devices have similar interstitial layout. However, it controls the parametrization of the ImageView's drawable resource performed until now in the layout xml

This PR also adds a new screenshot for the Quest Pro  controllers.

Fixes #446

### CODE REVIEW

File: app/src/oculusvr/res/layout/webxr_interstitial_controller.xml [hunk 1]
```diff
@@ -28,12 +28,12 @@
             android:layout_height="130dp"
             >
             <ImageView
+                android:id="@+id/left_controller"
```
Comment (svillar) [inline app/src/oculusvr/res/layout/webxr_interstitial_controller.xml:31]: This indeed should work but as I mentioned in the other comment I think it's better to do it differently.

Here you should be able to define a different RelativeLayout for the Meta Quest Pro (we select which one is shown with the condition in `app:visibleGone`. I'd basically copy everything and just modify the `app:visibleGone` and the `android:src` values for the case of the Meta Quest Pro.

**Review by svillar — CHANGES_REQUESTED** [review]

File: app/src/oculusvr/res/layout/webxr_interstitial_controller.xml (hunk 1 above)
Comment (javifernandez) [reply app/src/oculusvr/res/layout/webxr_interstitial_controller.xml]: Done

**Review by svillar — CHANGES_REQUESTED** [review]
Changes look great now. We just need higher quality images and also the left controller.

**Comment by javifernandez** [comment]
> Changes look great now. We just need higher quality images and also the left controller.

You mean adjusting the position of the left controller, right ? There is only need of one controller image, since the other is generated inverting the original one.

**Review by svillar — APPROVED** [review]
They look wonderful.

**Pull request merged**